app/src/main/java/com/keylesspalace/tusky/network/ProgressRequestBody.java

```diff
@@ -67,8 +67,10 @@ public void writeTo(@NonNull BufferedSink sink) throws IOException {
                 uploaded += read;
                 sink.write(buffer, 0, read);
             }
+
+            uploadListener.onProgressUpdate((int)(100 * uploaded / contentLength));
```

Comment:
This fills the upload progress circle.

(But maybe the incomplete circle was intended as there seems to be some processing after it before the blue circle actually disappears?)

Reply:
Maybe move this unrelated change to a separate PR?

Reply:
It's such a small collateral fix. :-)

I could remove it.